repos/kaleido/cc/scopes/Mermaid.h

```diff
@@ -0,0 +1,87 @@
+//
+// Created by coma007 on 4/4/24.
+//
+#include "Base.h"
+#include "base/bind.h"
+#include "base/command_line.h"
+#include "base/strings/string_util.h"
+#include "base/strings/stringprintf.h"
+#include "headless/public/devtools/domains/runtime.h"
+#include "../utils.h"
+#include <streambuf>
+#include <string>
+#include <sstream>
+#include <iostream>
+#include <fstream>
+
+#ifndef CHROMIUM_MERMAIDSCOPE_H
+#define CHROMIUM_MERMAIDSCOPE_H
+
+namespace kaleido {
+    namespace scopes {
+
+        class MermaidScope : public BaseScope {
+        public:
+            MermaidScope();
+
+            ~MermaidScope() override;
+
+            MermaidScope(const MermaidScope &v);
+
+            std::string ScopeName() override;
+
+            std::vector<std::unique_ptr<::headless::runtime::CallArgument>> BuildCallArguments() override;
+
+        };
+
+        MermaidScope::MermaidScope()  {
+
+            // Initialize mermaid object code
+            std::string mermaidInit = "mermaid.initialize( { startOnLoad: false, securityLevel: 'loose'} ); window.mermaid = mermaid;";
+
+            // Process mermaidjs
+            if (HasCommandLineSwitch("mermaidjs")) {
+                std::string mermaidjsArg = GetCommandLineSwitch("mermaidjs");
+
+                // Check if value is a URL
+                GURL mermaidjsUrl(mermaidjsArg);
+                if (mermaidjsUrl.is_valid()) {
+                    scriptTags.push_back(mermaidjsArg);
+                } else if (mermaidjsArg.find("import") != std::string::npos && mermaidjsArg.find("+esm") != std::string::npos) {
```

Comment:
How can "import" be contained in command line arguments?

Reply:
Just for my understanding - what is the expected usage format of mermaidjs argument?